odoo/addons/base/models/ir_model.py

```diff
@@ -719,13 +719,13 @@ def _check_depends(self):
                     field = model._fields.get(name)
                     if field is None:
                         raise UserError(_(
-                            'Unknown field "%(field)s" in dependency "%(dependency)s"',
+                            'Unknown field “%(field)s” in dependency “%(dependency)s”',
                             field=name,
                             dependency=seq.strip(),
                         ))
                     if index < last and not field.relational:
                         raise UserError(_(
-                            'Non-relational field "%(field)s" in dependency "%(dependency)s"',
+                            'Non-relational field “%(field)s” in dependency “%(dependency)s”',
```

Comment:
Unrelated diff?

Reply:
Indeed, I just thought it would be weird if just that particular error message does not have curly quotes, and it was changed last week in the placeholders PRs, so 🤷‍♂️